From 8268c245f1d04c4fa2468f5c6a97c8fa541e666b Mon Sep 17 00:00:00 2001
From: Andrew Wang <wang@apache.org>
Date: Thu, 12 May 2016 23:48:37 -0700
Subject: [PATCH 1576/2748] HADOOP-11180. Change log message "token.Token:
 Cannot find class for token kind kms-dt" to
 debug. Contributed by Yi Liu.

(cherry picked from commit 3bfd29949fef6d8fb5df4c6398e77cde320d8c2d)
(cherry picked from commit d6f6b99c83b0945c7d125a57650041d2e707ba0e)
(cherry picked from commit 5da54eb3d4e8914ad1934dc7c7c8bef0d8c3dbfb)

Change-Id: I0bef932126e1ab0c6b4447ef630edb605abe8531
---
 .../org/apache/hadoop/security/token/Token.java    |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/hadoop-common-project/hadoop-common/src/main/java/org/apache/hadoop/security/token/Token.java b/hadoop-common-project/hadoop-common/src/main/java/org/apache/hadoop/security/token/Token.java
index d36f96f..251a9b9 100644
--- a/hadoop-common-project/hadoop-common/src/main/java/org/apache/hadoop/security/token/Token.java
+++ b/hadoop-common-project/hadoop-common/src/main/java/org/apache/hadoop/security/token/Token.java
@@ -121,7 +121,7 @@ public Token(Token<T> other) {
       cls = tokenKindMap.get(kind);
     }
     if (cls == null) {
-      LOG.warn("Cannot find class for token kind " + kind);
+      LOG.debug("Cannot find class for token kind " + kind);
       return null;
     }
     return cls;
-- 
1.7.9.5

